Windows/PPSSPP.sln

```diff
@@ -75,18 +76,30 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "glslang", "..\ext\glslang.v
 EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug CRT DLL|Win32 = Debug CRT DLL|Win32
+		Debug CRT DLL|x64 = Debug CRT DLL|x64
 		Debug|Win32 = Debug|Win32
 		Debug|x64 = Debug|x64
+		Release CRT DLL|Win32 = Release CRT DLL|Win32
+		Release CRT DLL|x64 = Release CRT DLL|x64
```

Comment:
Do we want to add these, or were they just added by something else?

-[Unknown]